tools/trap/ramdumpParser.py

```diff
@@ -335,7 +335,7 @@ def read_string(self, address, format_string, debug=False):
 				print(('Failed to read address {0:x}'.format(addr)))
 			return None
 
-		# Unpack the string with proper format and return to calle.
+		# Unpack the string with proper format and return to caller
 		return struct.unpack(format_string, s)
 
 	# returns a word size (4 bytes = 32 bits) read from dump "<" means little endian format
```

Comment:
Can you pls make sure that all comments are starting with a capital letter so that we keep everything standardized?

Reply:
Done